server/src/instant/db/permissioned_transaction.clj

```diff
@@ -548,17 +553,18 @@
 
                 check-commands
                 (io/warn-io :check-commands
-                  (concat
-                   (attr-checks ctx attr-changes)
+                            (concat
```

Comment:
Do you know if there is some way to make conjure indent `io/warn-io` the same way it indents a `defn`?